.github/actions/release/action.yaml

```diff
@@ -0,0 +1,38 @@
+name: install-tools
+description: "Install pipeline tools"
```

Comment:
```suggestion
name: release
description: Given a module name, bump the version based upon the commit(s) since last release, generate release notes, tag with the bumped version, and trigger a GitHub release.
```